### DISCUSSION TITLE
Rename "RE" functions

### DISCUSSION THREAD

**Discussion by Denneisk** [wiremod/wire#2808] [Suggestions]
`findRE`, `matchRE`, `replaceRE`, and `explodeRE` are simply misleading. This form should be reserved for if actual regex ever gets implemented, and these functions should be superseded by `*Pattern` alternatives instead.

**Comment by Divran** [comment]
Maybe yes but I don't think renaming them now is worth it

**Reply by vurvdev** [reply]
People still do get confused by the names sometimes, I see no harm in using an e2function alias and tagging these as deprecated in favor of functions with new names

https://discord.com/channels/231131817640460288/231356264469233664/1123023231327158272

**Reply by vurvdev** [reply]
That or adding overloads that take a regex/pattern type
